samaritan/src/App.tsx

```diff
@@ -7,18 +7,9 @@ const App: React.FC = () => {
   return (
     <div className="App">
       <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
         <p>
-          Edit <code>src/App.tsx</code> and save to reload.
+          Samaritan Security
```

Comment:
do `git fetch` then `git pull origin master` i think there's a merge conflict from this so the branch can't be merged, then accept changes and should be okay